### DISCUSSION TITLE
Add support for Dependent Drop-down Lists

### DISCUSSION THREAD

**Discussion by nesscx** [exceljs/exceljs#2522] [Ideas & Proposals]
Or at least allow overwriting an excel that already has this functionality without damaging it.

<a href="https://www.excel-easy.com/examples/dependent-drop-down-lists.html">Here is an example of this feature</a>.

**Comment by javiermanzano** [comment]
is there any chance that this is possible with latest versions? I could help with a PR if needed

**Comment by vvldn** [comment]
What is the status of this proposal? Is someone actively working on this?
